grpc-server/grpc_server_test.go

```diff
@@ -71,3 +98,81 @@ func TestSetupGRPC(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCerts(t *testing.T) {
+	cases := []struct {
+		name      string
+		setupFunc func(t *testing.T) (string, error)
+		wanterr   error
+	}{
+		{
+			"Real key file",
+			func(t *testing.T) (string, error) {
+				t.Helper()
+				return "./testdata", nil
```

Comment:
I was questioning the addition of fixtures in this particular pattern. I see that we already have a precedent for text fixtures in a `testdata` directory (`db/testdata`). This is ok 👍